Source/Scene/PointCloud3DTileContent.js

```diff
@@ -96,7 +96,7 @@ Object.defineProperties(PointCloud3DTileContent.prototype, {
   batchTableByteLength: {
     get: function () {
       if (defined(this._batchTable)) {
-        return this._batchTable.memorySizeInBytes;
+        return this._batchTable.batchTableByteLength;
```

Comment:
This was one change I wasn't quite sure about. the old code sometimes uses `xxxByteLength` but also some things had `memorySizeInBytes`. I should try to make things more consistent.